subsys/bluetooth/shell/media_controller.c

```diff
@@ -127,24 +127,44 @@ static void track_duration_cb(struct media_player *plr, int err, int32_t duratio
 	shell_print(ctx_shell, "Player: %p, Track duration: %d", plr, duration);
 }
 
-static void track_position_cb(struct media_player *plr, int err, int32_t position)
+static void track_position_recv_cb(struct media_player *plr, int err, int32_t position)
 {
 	if (err) {
-		shell_error(ctx_shell, "Player: %p, Track position failed (%d)", plr, err);
+		shell_error(ctx_shell, "Player: %p, Track position receive failed (%d)", plr, err);
 		return;
 	}
 
-	shell_print(ctx_shell, "Player: %p, Track Position: %d", plr, position);
+	shell_print(ctx_shell, "Player: %p, Track Position received: %d", plr, position);
 }
 
-static void playback_speed_cb(struct media_player *plr, int err, int8_t speed)
+static void track_position_write_cb(struct media_player *plr, int err, int32_t position)
 {
 	if (err) {
-		shell_error(ctx_shell, "Player: %p, Playback speed failed (%d)", plr, err);
+		shell_error(ctx_shell, "Player: %p, Track position write failed (%d)", plr, err);
 		return;
 	}
 
-	shell_print(ctx_shell, "Player: %p, Playback speed: %d", plr, speed);
+	shell_print(ctx_shell, "Player: %p, Track Position write: %d", plr, position);
+}
+
+static void playback_speed_recv_cb(struct media_player *plr, int err, int8_t speed)
+{
+	if (err) {
+		shell_error(ctx_shell, "Player: %p, Playback speed receive failed (%d)", plr, err);
```

Comment:
Since you actually have added the attempted value in the callback, we might as well log those

Reply:
The same applies multiple places

Reply:
Since this is the shell, the user will usually just have entered that value, so it normally does not matter much.
Anyway, this is already existing code, and out of scope fo rthis PR.